authbridge/demos/github-issue/aiac/keycloak_ops/delete_policy.py

```
@@ -0,0 +1,88 @@
+        url = f"{admin.connection.base_url}/admin/realms/{realm}/roles-by-id/{realm_role['id']}/composites"
+        admin.connection.raw_delete(url, data=json.dumps(composite_roles))
+
+        composite_names = [
+            f"{(r.get('clientRole', False) and r.get('containerId', 'client')) or 'realm'}.{r['name']}"
+            for r in composite_roles
+        ]
+        print(
+            f"  ✓ Removed {len(composite_roles)} composite role(s) from "
+            f"'{realm_role_name}': {', '.join(composite_names)}"
+        )
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)